components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1647,6 +1652,26 @@ where
             return;
         }
 
+        // apply is ahead of raft, need to force raft commit to applied index.
```

Comment:
Can you please add a comment that explains under what circumstances, applied index could be larger than last commit?